controllers/controllers.go

```diff
@@ -88,3 +90,36 @@ func checkCSRF(r *http.Request, token string) bool {
 	}
 	return true
 }
+
+func BadRequest(w http.ResponseWriter, r *http.Request) {
+	tpl, err := pongo2.DefaultSet.FromFile("400.html.tpl")
+	if err != nil {
+		logging.SharedInstance().MethodInfo("Controllers", "BadRequest", true).Errorf("template error: %v", err)
+		http.Error(w, "400 BadRequest", 400)
+		return
+	}
+	tpl.ExecuteWriter(pongo2.Context{"title": "BadRequest"}, w)
+	return
+}
+
+func NotFound(w http.ResponseWriter, r *http.Request) {
+	tpl, err := pongo2.DefaultSet.FromFile("404.html.tpl")
+	if err != nil {
+		logging.SharedInstance().MethodInfo("Controllers", "NotFound", true).Errorf("template error: %v", err)
+		http.Error(w, "404 NotFound", 404)
+		return
+	}
+	tpl.ExecuteWriter(pongo2.Context{"title": "NotFound"}, w)
+	return
+}
+
+func InternalServerError(w http.ResponseWriter, r *http.Request) {
```

Comment:
__[golint]__
* [golint] exported function InternalServerError should have comment or be unexported